jenkins-slaves/ansible/data/config/bashrc

```diff
@@ -8,8 +8,9 @@ fi
 export LANG=en_US.UTF-8
 export JAVA_HOME=/opt/tools/jdk1.8
 export MAVEN_HOME=/opt/tools/apache-maven-3.6.3
+export OPERATORSDK_HOME=/opt/tools/extras/operator-sdk
 export MAVEN_OPTS="-Xmx2g"
 export GRAALVM_HOME=/opt/tools/graalvm
 export GOROOT=/opt/tools/go
 export GOPATH=/home/jenkins/go
-export PATH=$JAVA_HOME/bin:$MAVEN_HOME/bin:$GOROOT/bin:$GRAALVM_HOME/bin:$GOPATH/bin:/opt/tools/mercurial/:/opt/tools/extras/:$PATH
\ No newline at end of file
+export PATH=$JAVA_HOME/bin:$MAVEN_HOME/bin:$GOROOT/bin:$OPERATORSDK_HOME/v1.2.0:$GRAALVM_HOME/bin:$GOPATH/bin:/opt/tools/mercurial/:/opt/tools/extras/:$PATH
```

Comment:
I would add `v0.18.2`, wouldn't you?

Reply:
1.2.0 should be default now, since 1.1 is done now(only waiting for operatorHub Guys). So we need version according to current master :)